sys/net/include/ieee802154_frame.h

```diff
@@ -8,19 +9,20 @@
 
 /**
  * @defgroup    net_ieee802154 IEEE802.15.4
- * @ingroup net
- * @brief   IEEE802.15.4 adapaption layer
+ * @ingroup     net
+ * @brief       IEEE802.15.4 adapaption layer
  * @{
  *
  * @file
- * @brief   IEEE 802.14.4 framing data structs and prototypes
+ * @brief       IEEE 802.14.4 framing data structs and prototypes
  *
- * @author  Stephan Zeisberg <zeisberg@mi.fu-berlin.de>
- * @author  Oliver Hahm <oliver.hahm@inria.fr>
+ * @author      Stephan Zeisberg <zeisberg@mi.fu-berlin.de>
+ * @author      Oliver Hahm <oliver.hahm@inria.fr>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
  */
 
-#ifndef IEEE802154_IEEE802154_FRAME
-#define IEEE802154_IEEE802154_FRAME
+#ifndef __IEEE802154_FRAME_H
+#define __IEEE802154_FRAME_H
```

Comment:
Hm, good to know, I actually use the `__SOMETHING_H` template since I started programming C and I never really questioned it... Will fix this - and I guess we should take this into our coding conventions.